Documentation/docs/learn/third_party_applications.md

```diff
@@ -0,0 +1,100 @@
+
+- [**DeVIDE**](https://graphics.tudelft.nl/devide/) is the Delft
+  Visualization Development Environment, a cross-platform software
+  framework that provides infrastructure for the rapid creation,
+  testing and application of modular image processing and
+  visualisation algorithm implementations.
+- [**MeVisLab**](https://www.mevislab.de/) is a modular framework
+  for image processing research and development with a special
+  focus on medical imaging.
+- [**SCIRun**](https://www.sci.utah.edu/cibc-software/scirun.html) is a
+  Problem Solving Environment (PSE), for modeling, simulation and
+  visualization of scientific problems.
+- [VisTrails**](https://www.vistrails.org//index.php/Main_Page) is an
+  open-source scientific workflow and provenance management system
+  that supports data exploration and visualization.
+- [**Voreen**](https://www.uni-muenster.de/Voreen/) is an open-source
+  rapid application development framework for the interactive
+  visualization and analysis of multi-modal volumetric data sets.
+
+### Frameworks
+
+- [**Elastix**](https://elastix.lumc.nl/) is a toolbox for rigid and
+  nonrigid registration of images.
+- [**GIMIAS**](https://link.springer.com/chapter/10.1007/978-3-642-01932-6_45) was
+  an open-source framework for efficient development of research tools
+  and clinical prototypes.
+- [**MARVIN**](https://www.sciencedirect.com/science/article/abs/pii/S0169260708000965)
+  was a C++ application framework for rapid and simple development of medical
+  applications, part of [NCCR "Co-Me - Computer Aided and Image Guided Medical Interventions"](https://www.snf.ch/en/9ihBpawyw4Fro4Y9/page/researchinFocus/nccr/nccr-co-me/).
+- [**MITK**](https://www.mitk.org/wiki/The_Medical_Imaging_Interaction_Toolkit_(MITK))
+  is a free open-source software for the development of interactive
+  medical image processing software.
+- [**IGSTK**](https://github.com/Kitware/IGSTK) was a free open-source
+  toolkit for development of image guided interventions software.
+- [**Orfeo Toolbox**](https://www.orfeo-toolbox.org/) is an open-source
+  project for state-of-the-art remote sensing.
+
+### Packages
+
+- [**Python(x,y)**](https://python-xy.github.io/) is a
+  scientific-oriented Python distribution, including ITK, VTK, and Qt.
+
+### Utilities/helpers
+
+- [**MATITK**](https://www.mathworks.com/matlabcentral/fileexchange/12297-matitk)
+  was a MATLAB interface to ITK. See also on [MW File
+  exchange](http://www.mathworks.com/matlabcentral/fileexchange/12297)
+- [**MATLAB ITK Import**](https://github.com/thewtex/matlab-itk-import/tree/master)
+  allows to import any file supported by ITK into The Mathworks MATLAB. See also
+  the [CMake MatlabMex](https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/MatlabMex)
+  documentation.
+- [**Insight Applications**](https://itk.org/Wiki/ITK/Insight_Applications)
+- [**Command line wrappers of ITK functionality**](https://itk.org/Wiki/ITK/CommandLineTools)
```

Comment:
In terms of ITK command line tool, notable, maintained ones to point are the c3d command line tools developed by @pyushkevich : http://www.itksnap.org/pmwiki/pmwiki.php?n=Documentation.CommandLine

and the CLI applications developed by @aylward in the ITKTubeTK repository: https://github.com/InsightSoftwareConsortium/ITKTubeTK .

Reply:
Added/changed in 8d9650f.